llvm/test/Transforms/InstCombine/shift-logic.ll

```diff
@@ -240,6 +240,35 @@ define i32 @lshr_or_extra_use(i32 %x, i32 %y, ptr %p) {
   ret i32 %sh1
 }
 
+define i32 @lshr_or_extra_use_multiuse(i32 %x, ptr %p) {
```

Comment:
```suggestion
define i32 @lshr_or_extra_use_shift(i32 %x, ptr %p) {
```
Or something. extra_use and multiuse are the same thing, so using both makes no sense.